wiki/Tournaments/3WC/2021/en.md

```diff
@@ -182,6 +223,28 @@ The 3 Digit World Cup is run by various community members.
 
 ## Match results
 
+### Quarterfinals
+
+The stats of the Quarterfinals can be found [here]("https://docs.google.com/spreadsheets/d/146pzh4VK_PuUh2k-AT8T8IRpNWt_da6NmrdTiU0eZR4/edit#gid=1091288063" "Google spreadsheet").
```

Comment:
please no `here`s, every link has a title, or a generalized meaning you can pass to a reader
```suggestion
The stats of the Quarterfinals can be found [in the spreadsheet]("https://docs.google.com/spreadsheets/d/146pzh4VK_PuUh2k-AT8T8IRpNWt_da6NmrdTiU0eZR4/edit#gid=1091288063" "Google spreadsheet").
```